var/spack/repos/builtin/packages/py-ocp-models/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class PyOcpModels(PythonPackage):
+    """Reference implementation for the MLPerf HPC OpenCatalyst DimeNet++ benchmark"""
+
+    homepage = "https://opencatalystproject.org/"
+    git = "https://github.com/mlcommons/hpc.git"
+
+    version("main", branch="main")
+
+    tags = ["proxy-app"]
+
+    depends_on("python@3.8:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.8", type=("build", "run"))
```

Reply:
Did spack specs and constraints change recently? I would think this would restrict it to only a specific version of python? But changed to match conventions

Reply:
3.8 means 3.8.*

This didn't used to work with the old concretizer, but it works fine with clingo now.

Reply:
Ah. Awesome. That will teach me for never reading the documentation

Reply:
It's poorly documented at best lol.